src/app/filecoin-plus/allocators/components/SelectFilter.tsx

```diff
@@ -45,11 +45,7 @@ export function SelectFilter({
 
   return (
     <HeadlessUIListbox value={selectedOption} onChange={handleOptionChange}>
-      <HeadlessUIListboxButton className="inline-flex w-full items-center justify-between gap-2 rounded-lg border border-brand-300 bg-brand-800 p-3 text-brand-300 focus:brand-outline hover:border-current hover:text-brand-400">
-        <span className="truncate">{selectedOption.name}</span>
-        <Icon component={CaretDown} size={16} weight="bold" />
-      </HeadlessUIListboxButton>
-
+      <ListboxButton text={selectedOption.name} />
```

Comment:
Should be compact below `sm` - Fixed it in CR.

Now
![CleanShot 2024-11-08 at 13 19 21@2x](https://github.com/user-attachments/assets/53f90490-ce80-46cc-815d-1fe9be543b

Before
![CleanShot 2024-11-08 at 13 20 00@2x](https://github.com/user-attachments/assets/c8feb286-efde-41a2-8244-798ab2e519e1)
bb)